cpp/src/parquet/arrow/path_internal.h

```diff
@@ -115,16 +116,41 @@ class PARQUET_EXPORT MultipathLevelBuilder {
   ///   the elements in the top level array nullable.
   /// \param[in, out] context for use when allocating memory, etc.
   /// \param[out] write_leaf_callback Callback to receive results.
-  /// There will be one call to the write_leaf_callback for
+  /// There will be one call to the write_leaf_callback for each leaf node.
   static ::arrow::Status Write(const ::arrow::Array& array, bool array_nullable,
```

Comment:
Could you explain this parameter more? It seems like it should always be equivalent to `array.null_count() == 0`

Reply:
renamed and added some commentary.  This determines if the field indicates its nullable.